### PR TITLE
Merging internal commits for release/6.0.1xx

### PR DESCRIPTION
**Pull request by vseanreesermsft**

### CODE REVIEW

**Comment by marcpopMSFT** [comment]
We need to remove the dotnet6-internal feed from all of these PRs as that was added to enabled the nuget fix codeflow for Feb. Once removed, we need to ensure that the nuget packages shipped today are on the correct public feeds (they don't always get correctly mirrored as nuget doesn't frequently ship in servicing)

File: NuGet.config [hunk 1]
```diff
@@ -4,32 +4,28 @@
...
+    <add key="darc-int-dotnet-windowsdesktop-752aa75" value="https://pkgs.dev.azure.com/dnceng/internal/_packaging/darc-int-dotnet-windowsdesktop-752aa75c/nuget/v3/index.json" />
     <!--  End: Package sources from dotnet-windowsdesktop -->
     <!--End: Package sources managed by Dependency Flow automation. Do not edit the sources above.-->
     <add key="dotnet6" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet6/nuget/v3/index.json" />
     <add key="dotnet6-transport" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet6-transport/nuget/v3/index.json" />
     <add key="dotnet-public" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-public/nuget/v3/index.json" />
     <add key="dotnet-eng" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-eng/nuget/v3/index.json" />
     <add key="dotnet-tools" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet-tools/nuget/v3/index.json" />
+    <add key="dotnet6-internal" value="https://pkgs.dev.azure.com/dnceng/internal/_packaging/dotnet6-internal/nuget/v3/index.json" />
```
Comment (nagilson) [inline NuGet.config:28]:
```suggestion

```

**Review by nagilson — APPROVED** [review]

**Comment by nagilson** [comment]
this is likely still going to be broken until nuget packages are pushed  https://github.com/dotnet/installer/pull/18645

**Pull request merged**